js/logic.js

```diff
@@ -12,6 +12,7 @@ const logic = (function () {
   let currTime;
   let cdCounter;
   let cdCounterGoal;
+  let endTime;
```

Comment:
Unused variable here, `estimateTime` function re-defines the variable within the function scope.


```suggestion
```